### DISCUSSION TITLE
Regarding collision of rigid bodies.

### DISCUSSION THREAD

**Discussion by fluidkocy** [InteractiveComputerGraphics/SPlisHSPlasH#266] [Q&A] [answered]
Where can I find documentation on "collisionObjectType"?
If you have it, please give me the url.
If not, please tell me the difference between the "collisionObjectType" numbers.

**Comment by janbender** [comment, marked as answer]
SPlisHSPlasH uses the library PositionBasedDynamics for the simulation of the rigid bodies. There you also find the documentation: 

https://positionbaseddynamics.readthedocs.io/en/latest/file_format.html#rigidbodies

PositionBasedDynamics also reads the same scene file and picks out all relevant information. Therefore, you can use all keys of the PositionBasedDynamics scene files.
